data/voiceovers/README.md

```diff
@@ -1,9 +1,11 @@
-Language accent master list stores a JSON with with language code as keys and nested dicts as values.
-Each nested dict contains language accent code as keys and its description as values.
-This is an exhaustive list of language accent pairs that Oppia may support for voiceovers (manual and auto).
-
-
-Autogeneratable language accent list stores a JSON with language accent codes as keys and nested dicts as values.
-Each nested dict contains 'service' and 'voice_code' as keys and their respective field values as values.
-This is an exhaustive list of language accent pairs that Oppia may support for automatic voiceovers, and
-this should be a subset of language accent master list.
+This folder contains two JSON files:
+    - language_accent_master_list.json stores a JSON with language codes as keys and nested dicts as values.
```

Comment:
Not sure what "a JSON" is. Do you mean a dict? Ditto below.

Reply:
Updated